src/smart-components/role/role-resource-definitions.js

```diff
@@ -0,0 +1,148 @@
+import React, { Fragment, useEffect, useState } from 'react';
+import { TextContent, Text, TextVariants, Level, LevelItem, Button } from '@patternfly/react-core';
+import { shallowEqual, useSelector, useDispatch } from 'react-redux';
+import { TableToolbarView } from '../../presentational-components/shared/table-toolbar-view';
+import { createRows } from './role-resource-definitions-table-helpers';
+import { useParams, Link, Route } from 'react-router-dom';
+import { TopToolbar } from '../../presentational-components/shared/top-toolbar';
+import { PageHeaderTitle } from '@redhat-cloud-services/frontend-components/components/PageHeader';
+import { ToolbarTitlePlaceholder } from '../../presentational-components/shared/loader-placeholders';
+import { defaultSettings } from '../../helpers/shared/pagination';
+import { fetchRole } from '../../redux/actions/role-actions';
+import { routes as paths } from '../../../package.json';
+import EditResourceDefinitionsModal from './edit-resource-definitions-modal';
+import './role-permissions.scss';
+
+const ResourceDefinitions = () => {
+  const [config, setConfig] = useState({
+    pagination: {
+      ...defaultSettings,
+      filter: '',
+    },
+  });
+
+  const { pagination, filter } = config;
+
+  const columns = [{ title: 'Title' }];
```

Comment:
Shouldn't be here some more specific string?

Reply:
> Shouldn't be here some more specific string?

I don't think it matters, the table header isn't visible. It just doesn't work for me when the name wasn't defined at all. Actually I can put empty object instead of this to columns